bib/go.mod

```diff
@@ -7,7 +7,8 @@ require (
 	github.com/cheggaaa/pb/v3 v3.1.7
 	github.com/hashicorp/go-version v1.7.0
 	github.com/mattn/go-isatty v0.0.20
-	github.com/osbuild/images v0.121.0
+	github.com/osbuild/blueprint v0.0.0-20250326072213-45708b085654
+	github.com/osbuild/images v0.127.0
```

Comment:
The dependency update pulled in a newer version of images, so we're going to have to merge #864 first.

Reply:
Thanks, I rebased and also pulled in the latest version of the blueprint lib too